cytotable/presets.py

```diff
@@ -204,6 +204,35 @@
                 AND nuclei.Nuclei_ObjectNumber = cytoplasm.Metadata_Cytoplasm_Parent_Nuclei
         """,
     },
+    "in-carta": {
+        # version specifications using related references
+        "CONFIG_SOURCE_VERSION": {
+            # note: uncertain version at this time, to be filled in as possible
+            "in-carta": "v",
+        },
+        # names of source table compartments (for ex. cells.csv, etc.)
+        "CONFIG_NAMES_COMPARTMENTS": tuple(),
+        # names of source table metadata (for ex. image.csv, etc.)
+        "CONFIG_NAMES_METADATA": tuple(),
```

Comment:
This seems premature to merge into main - maybe it's worth merging this into an in-carta-dev branch first?

Reply:
Thanks for this comment! I think you could be correct about the prematurity / uncertainty here. My instinct on the approach is to avoid dev branching, even in the face of uncertainty about this preset / the use within CytoTable. One perspective on this comes from a section on ["Dev Branches" from Ch. 16 of Software Engineering at Google](https://abseil.io/resources/swe-book/html/ch16.html#dev_branches).

To make sure we address this, I'll add another checkbox for #136 to validate that the compartment and metadata settings are accurate.